src/pages/api/featuredItem.ts

```diff
@@ -20,12 +20,23 @@ const featuredItemDataHandler = async (
       ? featuredImageID(request.query.imageID as string)
       : featuredImageID();
 
-    const dataFromUri = await getItemDataFromImageID(imageID);
+    const dataFromUri = await getFeaturedImage();
```

Comment:
nitpick: since we're not longer retrieving the featured image data via uri, it might make sense to change the name of this constant to something like "featuredImageData"

Reply:
oh yes, good idea!  thank you very much 🙌